packages/xod-func-tools/src/index.js

```diff
@@ -85,6 +85,29 @@ export const foldEither = def(
   )
 );
 
+/**
+ * Function to extract value from `Maybe` by providing
+ * a default value for `Nothing` and function to handle `Just`.
+ *
+ * This is hack to prevent errors on using `Maybe.maybe` with
+ * cross-package Maybies (`Maybe.maybe` checks for instanceof and
+ * it has other constuctors for some reason).
+ *
+ * @private
+ * @function foldMaybe
+ * @param {Function} nothingFn
+ * @param {Function} justFn
+ * @param {Maybe} maybeObject
+ * @returns {*}
+ * @throws Error
+ */
+export const foldMaybe = def(
+  'foldMaybe :: (() -> b) -> (a -> b) -> Maybe a -> b',
```

Comment:
why `(() -> b) -> (a -> b) -> Maybe a -> b` instead of `b -> (a -> b) -> Maybe a -> b`?

https://github.com/ramda/ramda-fantasy/blob/master/docs/Maybe.md#maybemaybe
https://pursuit.purescript.org/packages/purescript-maybe/3.0.0/docs/Data.Maybe#v:maybe
http://hackage.haskell.org/package/base-4.10.0.0/docs/Prelude.html#v:maybe

Reply:
Also,
```bash
cat ./packages/*/package.json | grep ramda-fantasy
    "ramda-fantasy": "^0.7.0",
    "ramda-fantasy": "^0.8.0",
    "ramda-fantasy": "^0.7.0",
    "ramda-fantasy": "^0.7.0",
    "ramda-fantasy": "^0.7.0",
```